scripts/cassandra_env.sh

```diff
@@ -0,0 +1,11 @@
+cd $(git rev-parse --show-toplevel)
```

Comment:
add a shebang ?

And a top-file comment to explain what it does and how to use it. Also applies to other file.

Reply:
Done.